src/Interpreters/Context.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <Access/EnabledRowPolicies.h>
```

Comment:
Context.h is included in so many source files, it doesn't seem this line is necessary.

Reply:
+

Reply:
Please check if it's okay, I removed it from here but added it to `Interpreters/InterpreterSelectQuery.h`, since `RowPolicyFilter` is a member of `InterpreterSelectQuery`. 73048d16cc